src/Twig/AssetRuntime.php

```diff
@@ -16,13 +16,17 @@ class AssetRuntime
     private $env;
     private $hash;
 
-    public function __construct(Router $router, BaseAssetExtension $symfonyAssetExtension, string $secret, string $env, string $hash)
+    public function __construct(Router $router, BaseAssetExtension $symfonyAssetExtension, string $secret, string $env, ?string $hash)
     {
         $this->router = $router;
         $this->symfonyAssetExtension = $symfonyAssetExtension;
         $this->secret = $secret;
         $this->env = $env;
         $this->hash = $hash;
+
+        if ('dev' !== $env && !$this->hash) {
```

Comment:
not fan of this.. it's just for deleting `assets_version.yml` file or for fixing a bug?

Reply:
Just to be sure prod env is configured with a proper hash since the default value is now `null`.

Reply:
It's a really bad idea to have an `assets_version.yml` with a real value in the repo.. It could hide some real problem. Like this it's less error prone at least.

Reply:
normally, the value of hash change on each build in production, isn't it?

Reply:
yeap